salt/modules/zypperpkg.py

```diff
@@ -539,14 +534,11 @@ def list_upgrades(refresh=True, root=None, **kwargs):
     cmd = ["list-updates"]
     if "fromrepo" in kwargs:
         repos = kwargs["fromrepo"]
-        if isinstance(repos, six.string_types):
+        if isinstance(repos, str):
             repos = [repos]
         for repo in repos:
             cmd.extend(
-                [
-                    "--repo",
-                    repo if isinstance(repo, six.string_types) else six.text_type(repo),
-                ]
+                [("--repo", repo) if isinstance(repo, str) else (str(repo), )]
```

Comment:
This needs to be `(str(repo), )` because I get a conflict between `pylint` and `black` otherwise. For consistency I've also added the parentheses to the first tuple.

`pylint` reports 
```
salt/modules/zypperpkg.py:541: [E8231(missing-whitespace-after-comma), ] PEP8 E231: missing whitespace after ','
```
but I can't add the whitespace -- `black` will remove it.

Reply:
ah no, remove the parenthesis:

```python
[("--repo", repo) if isinstance(repo, str) else str(repo)]
```

Reply:
Thanks, I've adjusted that now!